package.json

```diff
@@ -5,14 +5,15 @@
     "build": "turbo build",
     "bump": "depsynky bump && pnpm -w install",
     "test": "pnpm vitest --run",
-    "check:type": "pnpm -r --stream check:type",
+    "check:bplint": "pnpm -r --stream check:bplint",
+    "check:dep": "depsynky check",
     "check:format": "prettier --check .",
-    "fix:format": "prettier --write .",
     "check:lint": "eslint ./ --ext .ts --ext .tsx",
-    "fix:lint": "eslint --fix ./ --ext .ts --ext .tsx",
-    "check:dep": "depsynky check",
+    "check:type": "pnpm -r --stream check:type",
     "fix:dep": "depsynky sync",
-    "check": "pnpm check:dep && pnpm check:format && pnpm check:lint && pnpm check:type",
+    "fix:format": "prettier --write .",
```

Comment:
**nitpick:** I think I would do something like
```json
"check:bplint": "pnpm -r --stream check:bplint",
"check:eslint": "pnpm -r --stream check:bplint",
"check:lint": "pnpm check:eslint && pnpm run check:bplint"
```

Reply:
Instead of `check:bplint`, we could go with a more verbose `check:integrationlint` or `check:definition`... I'm not quite sure what terminology would be best